src/libraries/Microsoft.Extensions.Hosting/tests/UnitTests/HostTests.cs

```diff
@@ -226,19 +235,18 @@ public async Task CreateDefaultBuilder_SecretsDoesReload()
             var secretId = Assembly.GetExecutingAssembly().GetName().Name;
             var reloadFlagConfig = new Dictionary<string, string>() { { "hostbuilder:reloadConfigOnChange", "true" } };
             var secretPath = PathHelper.GetSecretsPathFromSecretsId(secretId);
-
             var secretFileInfo = new FileInfo(secretPath);
+
             Directory.CreateDirectory(secretFileInfo.Directory.FullName);
 
-            string SaveRandomSecret()
+            static string SaveRandomSecret(string secretPath)
             {
                 var newMessage = $"Hello ASP.NET Core: {Guid.NewGuid():N}";
                 File.WriteAllText(secretPath, $"{{ \"Hello\": \"{newMessage}\" }}");
                 return newMessage;
             }
 
-            string dynamicSecretMessage1 = SaveRandomSecret();
-
+            var dynamicSecretMessage1 = SaveRandomSecret(secretPath);
```

Comment:
Our style guide is:

10. We only use var when the type is explicitly named on the right-hand side, typically due to either new or an explicit cast, e.g. `var stream = new FileStream(...)` not `var stream = OpenStandardInput()`.

[https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md](https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md)